lib/public/views/lhcPeriods/overview/LhcPeriodsModel.js

```diff
@@ -0,0 +1,117 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { Observable, RemoteData } from '/js/src/index.js';
+import { getRemoteDataSlice } from '../../../utilities/fetch/getRemoteDataSlice.js';
+import { PaginationModel } from '../../../components/Pagination/PaginationModel.js';
+
+/**
+ * Model representing handlers for lhc periods page
```

Comment:
Looks like this is the comment that is on some other models, but tbh I think it's really bad. Maybe something like `Model for the LHC period pageS` would be enough.
Aren't you mixing periods and periods statistics again?

Reply:
applied